routes/index.js

```diff
@@ -64,7 +64,7 @@ module.exports = (keystone) => {
 				return next();
 			}
 
-			res.render('job', job);
+			res.redirect(job.url_base);
```

Comment:
Just pushed some more changes. The job page is back for all jobs, but I've replaced a bunch of fixed fields with one freeform one instead.